common/src/main/java/org/opensearch/ml/common/FunctionName.java

```diff
@@ -35,9 +39,12 @@ public static FunctionName from(String value) {
      * @return true for deep learning model.
      */
     public static boolean isDLModel(FunctionName functionName) {
-        if (functionName == TEXT_EMBEDDING || functionName == SPARSE_ENCODING || functionName == SPARSE_TOKENIZE) {
-            return true;
-        }
-        return false;
+        Set<FunctionName> dlmodels = new HashSet<>(Set.of(
```

Comment:
Create a static constant for DL models? No need to create a new HashSet for each `isDLModel` call.

Reply:
ack